connector_ecommerce/models/product.py

```diff
@@ -29,19 +12,23 @@ class ProductTemplate(models.Model):
 
     # TODO implement set function and also support multi tax
     @api.one
-    @api.depends('taxes_id', 'taxes_id.group_id')
-    def _get_tax_group_id(self):
+    @api.depends('taxes_id', 'taxes_id.tax_group_id')
```

Comment:
@api.depends('taxes_id', 'taxes_id.tax_group_id') **is equal to** @api.depends('taxes_id.tax_group_id')

So, we could remove taxes_id in depends.